internal/test/wmcb/pkginfo.go

```diff
@@ -52,31 +52,11 @@ type pkgInfo struct {
 // pkgInfoFactory returns PkgInfo specific to the package name
 func pkgInfoFactory(name pkgName, shaType string, baseUrl string, version string) (PkgInfo, error) {
 	switch name {
-	case cniPluginPkgName:
```

Comment:
is this file pkginfo required anymore?

Reply:
If we are removing the last user of `pkgInfo`, then `pkginfo.go` should be deleted using `git rm`.